playbooks/common/openshift-cluster/upgrades/files/ensure_system_units_have_version.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+
+SERVICE_TYPE=$1
+DEPLOYMENT_TYPE=$2
+VERSION="v${3}"
+
+add_image_version_to_sysconfig () {
```

Comment:
the lineinfile ansible module can do this (add a line only if the regex does not match)